cogboard-webapp/src/constants/index.js

```diff
@@ -129,5 +129,10 @@ export const NOTIFICATIONS = {
     type: 'info',
     message: `${capitalize(userRole)} logged out`,
     duration: 3000
+  }),
+  LOGOUT_REASON: (userRole, reason) => ({
```

Comment:
I think you can generalise above function to take reason arg ant set it's default value to empty string, so if you won't pass it to a function it will prints "" in template string. It is no point to keep 2 function doing the same job in 90% ;)